view/frontend/web/js/view/pdp/pdp-fast-button.js

```diff
@@ -67,20 +65,14 @@ define(['uiComponent', 'jquery', 'ko', 'underscore', 'fastConfig'],
                     }
```

Comment:
One request from Ruroc was to remove css override logic in here. see if just removing these lines will work properly on the front end

Reply:
Thanks, I missed that one. It's now updated.